src/funfuzz/ccoverage/__init__.py

```diff
@@ -0,0 +1,11 @@
+# coding=utf-8
+# flake8: noqa
+# pylint: disable=missing-docstring
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+from . import get_build
```

Comment:
This doesn't have any effect, does it?

Reply:
I reckoned this is the same as the functions in other places, for the whole Python packaging thing to work well. Having it does no harm, fwiw.

Reply:
Yeah, I just mean you're importing a file from `.` and not using it. At the module level, you could already import the file as `funfuzz.ccoverage.get_build` so at best it has no effect and at worst it's confusing. ;)

Reply:
There's a need for it:

```
$ pip install --user --upgrade -qqq ~/funfuzz/
$ python
Python 2.7.14 (default, Sep 18 2017, 11:31:52) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import funfuzz
>>> dir(funfuzz.ccoverage.get_build)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'get_build'
>>> 
$ echo 'from . import get_build' >> __init__.py
$ pip install --user --upgrade -qqq ~/funfuzz/
$ python
Python 2.7.14 (default, Sep 18 2017, 11:31:52) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import funfuzz
>>> dir(funfuzz.ccoverage.get_build)
['RUN_COV_LOG', '__builtins__', '__doc__', '__file__', '__name__', '__package__', 'absolute_import', 'division', 'get_coverage_build', 'get_grcov', 'io', 'logging', 'os', 'requests', 'tarfile', 'unicode_literals', 'zipfile']
>>> 
```

As you can see, without the line, pip/python is unable to call `funfuzz.ccoverage.get_build`, whereas with the line added, it is able to.

Well, unless this way is not the right one, I'll be open to other alternatives. :)